.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,12 @@
+<!--
+
+Thanks for creating this pull request!
+
+Please make sure to link the issue you are closing as "Closes #issueNr". 
+This helps us to understand the context of this PR.
+
+-->
+
+Closes #
+
+- [ ] This PR adds a new `form-js` element or visually changes an existing component.
```

Comment:
Maybe we can state an actionable item once this is met? E.g. 

```md
- [ ] This PR adds a new `form-js` element or visually changes an existing component.
  * => In that case, we need to ensure we follow up on this, e.g. by [creating an issue in Tasklist](https://github.com/camunda/tasklist/issues/new/choose)
```

Reply:
Done, finally managed to push after some hours because of the Github outage :sweat_smile: